.github/workflows/plugin_install.yml

```diff
@@ -3,7 +3,7 @@ name: Plugin Install
 on: [push, pull_request, workflow_dispatch]
 
 env:
-  OPENSEARCH_VERSION: 2.13.0
+  OPENSEARCH_VERSION: 2.14.0
```

Comment:
We would need to OpenSearch core first, distributions won't be available till than

Reply:
Sure, I was opening this because that is the order that the release issue says to do things in.